admin/src/react-components/account-edit-toolbar.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { Toolbar, SaveButton } from "react-admin";
+import { withStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { Dialog, DialogContent, DialogContentText, DialogActions } from "@material-ui/core";
+
+const accountEditToolbarStyles = {
```

Comment:
not a fan of this but looks like this is how all the components work. I think this project just need a lot of overall help.